main/communication.py

```diff
@@ -12,7 +12,8 @@ def file_picker():
     root.focus_force()  # Force focus
     file_path = filedialog.askopenfilename(
         title="Select a File",
-        filetypes=(("Text files", "*.txt"), ("All files", "*.*")),
+        filetypes=(("Text files", "*.txt"), ("JSON files", "*.json"), ("LOG files", "*.log"), ("DOCX files", "*.docx"),
+                   ("PDF files", "*.pdf"), ("XML files", "*.xml"), ("YML files", "*.yml"), ("All files", "*.*")),
     )
```

Comment:
I don't get it, are lines15, 16 allow to read extra formats,
If yes what for we need new functions as json_file_picker etc??

Reply:
Because You don't have extra functions in Notes page information that you added. And turn that Notes info into doc about the work you done or add it to communication module documentation.